app/src/util/networks.ts

```diff
@@ -201,6 +223,13 @@ const networks: { [K in NetworkId]: Network } = {
   [networkIds.XDAI]: {
     label: 'xDai',
     url: 'https://rpc.xdaichain.com/',
+    alternativeUrls: [
+      {
+        rpcUrl: 'https://rpc.xdaichain.com/',
+        name: 'xDai',
+      },
+      { rpcUrl: 'https://xdai-archive.blockscout.com', name: 'xDai' },
+    ],
```

Comment:
These should have different names to be less confusing. Maybe 'xDai' and 'xDai - Blockscout' or something like that